### PR TITLE
App launch time new docs

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - New Feature Doc and Intro Page for app launch time

**Question for doc team:**
1. Could you help to enrich the intro page wording if you see fit? These two are my more reference:

- https://developer.android.com/topic/performance/vitals/launch-time
- https://blog.instabug.com/understanding-cold-hot-and-warm-app-launch-time/

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Introduction to app launch times
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Get started
+metaDescription: Use New Relic's mobile monitoring capabilities to manage the performance of your iOS and Android apps and your customers' overall digital experience.
+---
+
+New Relic's mobile monitoring capabilities help you gain deeper visibility into how to analyze your [Android](/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android) and [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/introduction-new-relic-mobile-ios) application performance and troubleshoot crashes. With the abundance of high-quality apps out there, users have become very sensitive to app launch time and have very high expectations.
```
Comment (homelessbirds) [inline src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx:10]: @ying-nr why would a user need app launch times? what's the goal they're trying to solve by analyzing app launch times? Just trying to get to the heart of the user story.

File: src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Why we track mobile monitoring [#features]
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx:12]: Resolved

**Comment by ywang-nr** [comment]
@cthomas-newrelic @cdillard-NewRelic for feedbacks

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/with-launch-activity-name.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Set the target activity name for app launch time to calculate the cold time and hot time
+
+## Requirements
+
+Agent version 6.9.0 or higher.
+
+## Description
+
+By default, the New Relic Android agent calculates the cold time and hot time when the default activity loaded, this api provides the ability to customize the targe launch activity name when calculates the cold time and hot time.
```
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/with-launch-activity-name.mdx:28]: Capitalize API, as is done else where, or just make every instance consistent.

**Review by cthomas-newrelic — DISMISSED** [review]
Minor comments, best left to the docs experts

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/with-launch-activity-name.mdx (hunk 3 above)
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/with-launch-activity-name.mdx]: fixed

File: src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Get started
+metaDescription: Use New Relic's mobile monitoring capabilities to manage the performance of your iOS and Android apps and your customers' overall digital experience.
+---
+
+New Relic's mobile monitoring capabilities help you gain deeper visibility into how to analyze your [Android](/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android) and [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/introduction-new-relic-mobile-ios) application performance and troubleshoot crashes. With the abundance of high-quality apps out there, users have become very sensitive to app launch time and have very high expectations.
+
+## Why we track app launch time [#features]
```
[the remark this answers is not in the document]
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx]: fixed

File: src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+According to Google, your app’s startup time is not considered excessive if they fall within these thresholds:
+
+* Cold startup should take less than 5 seconds
+* Hot startup should take less than 1.5 seconds
+
+According to Apple, your app’s startup time is not considered excessive if they fall within 400ms. The app must launch in less than 2o seconds or the system will kill the app.
+
+
+## Configurations
+You'll be able to configure mobile app launch times for [Android](docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps)and iOS.
```
Comment (rhetoric101) [inline src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx]: @ywang-nr how do iOS users configure launch time?

**Review by rhetoric101 — COMMENTED** [review]
@ywang-nr Thanks for submitting these new docs! I made some simple edits that you should take a look at to make sure this still has your original intent. I inserted a question below.

File: src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+According to Google, your app's startup time is not considered excessive if it falls within these thresholds:
+
+* Cold startup should take less than 5 seconds
+* Hot startup should take less than 1.5 seconds
+
+According to Apple, your app's startup time is not considered excessive if it falls within 400ms. The app must launch in less than 20 seconds or the system will kill the app.
+
+## Configurations [#configurations]
+
+You you can configure mobile app launch times for [Android](docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps)and iOS.
```
Comment (ywang-nr) [inline src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx:37]: Removing the duplicate "you"

File: src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx (hunk 5 above)
Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx]: @rhetoric101 Thanks for the edits, they look great and I made one adjustment. IOS is not released yet, so we don't have a doc yet, can we add that later when it's ready?

**Review by rhetoric101 — APPROVED** [review]
@ywang-nr thanks for adding these docs!

**Pull request merged**